modules/apps/commerce/commerce-product-content-search-web/src/main/resources/META-INF/resources/specification_option_facets/view.jsp

```diff
@@ -38,7 +38,7 @@ CPSpecificationOptionFacetsDisplayContext cpSpecificationOptionFacetsDisplayCont
 
 			String parameterName = cpSpecificationOptionsSearchFacetDisplayContext.getParameterName();
 
-			String panelTitle = HtmlUtil.escape(cpSpecificationOptionsSearchFacetDisplayContext.getCPSpecificationOptionTitle(facet.getFieldName()));
+			String panelTitle = HtmlUtil.escapeAttribute(cpSpecificationOptionsSearchFacetDisplayContext.getCPSpecificationOptionTitle(facet.getFieldName()));
```

Comment:
this way you are also changing the panel title, it was explicitly requested to not change that, because it was right.

please inline this HtmlUtil.escapeAttribute(cpSpecificationOptionsSearchFacetDisplayContext.getCPSpecificationOptionTitle(facet.getFieldName())); in the attribute value